meson_options.txt

```diff
@@ -1,7 +1,7 @@
 option('build', type : 'combo', choices : ['all', 'standalone', 'library'], value : 'all', description : 'build type')
 option('consolekit', type : 'feature', description : 'ConsoleKit support', deprecated: {'true': 'enabled', 'false': 'disabled'})
 option('static_analysis', type : 'boolean', value : false, description : 'enable GCC static analysis support')
-option('gresource_quirks', type : 'feature', description : 'compile the quirk files into the fwupd binary')
+option('compressed_quirks', type : 'boolean', description : 'compile the quirk files into a compressed blob')
```

Comment:
I think this should stay as a feature. It can be auto enabled when gzip is found.

Reply:
My thinking here is that the compressed_quirks might make it slightly harder to do development (as in you have to run from a checkout, rather than just dumping files into quirks.d to *replace* the existing quirk data -- maybe. In the common case (adding an extra VID/PID) you can still add quirks as before. As in it's not so much related to the availability of gzip as to the behaviour the system builder wanted.

Maybe we just rip off the sticky plaster and make compressed_quirks the only way to do it? i.e. if there's a (in my head, theoretical) problem with overriding compressed builtin data then we fix the bug then and just don't have an option to disable it. Certainly KISS is easier to debug... Certainly using one compressed file is two order of magnitude less space for quirks on disk, as at the moment even the smallest 12 byte quirk files still take up one block (e.g.4kb) on disk, and 100x4kb is kind big.

Reply:
I've made it simpler.